lib/components/Signature.js

```diff
@@ -182,15 +203,25 @@ export default class Signature extends React.Component<Props> {
   };
 
   render() {
-    return (
-      <GLView
-        {...this.panResponder.panHandlers}
-        onLayout={this.onLayout}
-        key={'Expo.Signature-' + global.__ExpoSignatureId}
-        ref={this.setRef}
-        {...this.props}
-        onContextCreate={this.onContextCreate}
+    return ( <
+      GLView {
+        ...this.panResponder.panHandlers
+      }
+      onLayout = {
+        this.onLayout
+      }
+      key = {
+        'Expo.Signature-' + global.__ExpoSignatureId
+      }
+      ref = {
+        this.setRef
+      } {
```

Comment:
This is a terrible place to break a line - `ref={this.setRef}` and `{...this.props}` are two entirely different props.  Technically `{...this.props}` is a set of zero of more props, but it's still not `ref`.

This entire block hasn't changed - why are you messing with the layout of it?  All that's doing is making it harder to read for the person who wrote it in the first place.